### PR TITLE
Dashboard Layout

### PR DESCRIPTION
**Pull request by Vanessa-Kris**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+  </body>
+</html>
```
Comment (PromiseFru) [inline public/index.html:43]: Update the content, remove irrelevant comments, and ensure the correct content is used.

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```
Comment (PromiseFru) [inline public/manifest.json:25]: Update the content of the manifest to ensure that the correct information is included.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```
Comment (PromiseFru) [inline src/index.js:16]: Remove irrelevant comments

File: src/logo.svg
Comment (PromiseFru) [inline src/logo.svg:1]: Remove irrelevant files

**Review by PromiseFru — CHANGES_REQUESTED** [review]
Please see my comments, @Vanessa-Kris.

**Comment by Vanessa-Kris** [comment]
@PromiseFru I've made the required update/changes.

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Gateway Clients Resiliance Dashboard"
```
Comment (PromiseFru) [inline public/index.html:10]:
```suggestion
      content="Gateway Clients Reliability Dashboard"
```

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+   
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+   
+    <title>Dashboard</title>
```
Comment (PromiseFru) [inline public/index.html:16]:
```suggestion
    <title>Gateway Clients Reliability Dashboard</title>
```

File: public/manifest.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Resiliance Dashboard",
+  "name": "Gateway Clients Resiliance Dashboard",
```
Comment (PromiseFru) [inline public/manifest.json:3]:
```suggestion
  "short_name": "Reliability Dashboard",
  "name": "Gateway Clients Reliability Dashboard",
```

**Review by PromiseFru — CHANGES_REQUESTED** [review]
Looking good, @Vanessa-Kris! Just a few more minor tweaks. 😉

File: src/App.css [hunk 7]
```diff
@@ -1,13 +1,26 @@
...
+
+* {
+  font-family: 'Suravaram', sans-serif !important;
 }
 
+/* body {
+  background-image: linear-gradient(to left, #170624 0%, #170624 0%, #1E0041 21%, #3B009F 52%, #170624 78%, #170624 100%);
+}
+
+
```
Comment (PromiseFru) [inline src/App.css:23]: Remove irrelevant comments

**Review by PromiseFru — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 7 above)
Comment (Vanessa-Kris) [reply src/App.css]: @PromiseFru  done.

**Comment by PromiseFru** [comment]
Thank you, @Vanessa-Kris! We could make the table responsive for mobile view and consider adding a back button to the test display page. 

image

image

**Review by PromiseFru — APPROVED** [review]
LGTM thank you @Vanessa-Kris!

**Comment by Vanessa-Kris** [comment]
Thank you! @PromiseFru  I've also added the back button

**Pull request merged**